google/cloud/bigtable/examples/README.md

```diff
@@ -1,208 +1,143 @@
-$ export ZONE=... # Name of the zone where the example will create a new instance
-$ ./bigtable_samples_instance_admin run ${PROJECT_ID} ${INSTANCE_ID} ${CLUSTER_ID} ${ZONE}
+$ ./table_admin_snippets create-table <project-id> <instance-id> <table-id>
 ```
 
-[hello_world_code]: bigtable_hello_world.cc
-[instance_admin_code]: bigtable_hello_instance_admin.cc
-
-### Administer Tables
-
-This sample showcases the basic table / column family operations:
+## Reading and Writing Data Operations
 
-- Create a table
-- List tables in the current project
-- Retrieve table metadata
-- List table column families and GC rules
-- Update a column family GC rule
-- Delete all the rows
-- Delete a table
+| Target                   | Description                                      |
+| ------------------------ | ------------------------------------------------ |
+| `./read_snippets`        | Collection of synchronous read operations        |
+| `./data_snippets`        | Collection of other synchronous table operations |
+| `./data_async_snippets`  | Collection of asynchronous table operations      |
+| `./data_filter_snippets` | Collection of operations using [Filters](https://cloud.google.com/bigtable/docs/filters)|
 
-#### Usage
-
-```console
-$ ./bigtable_samples
-
-Usage: bigtable_samples <command> <project_id> <instance_id> <table_id> [arguments]
-
-Examples:
-  bigtable_samples run my-project my-instance my-table
-  table_admin_snippets create-table my-project my-instance my-table
-  table_admin_snippets list-tables my-project my-instance
-  table_admin_snippets get-table my-project my-instance my-table
-  table_admin_snippets modify-table my-project my-instance my-table
-  table_admin_snippets drop-all-rows my-project my-instance my-table
-  table_admin_snippets delete-table my-project my-instance my-table
-```
-#### Run admin table samples
-After configuring gRPC, you can run the examples using:
+The above samples demonstrate individual data operations. Running the targets with
+the `--help` flag will display the available commands and their usage. Here is an example of one
+such command which will read a row from a table.
 
 ```console
-$ export PROJECT_ID=... # Your Google Cloud Platform project ID
-$ export INSTANCE_ID=... # Cloud Bigtable instance ID
-$ export TABLE_ID=... # Cloud Bigtable table ID
-$ ./bigtable_samples run ${PROJECT_ID} ${INSTANCE_ID} ${TABLE_ID}
+$ ./read_snippets read-row <project-id> <instance-id> <table-id> <row-key>
 ```
 
-### Reading and Writing Operations
+## Credentials
 
-This sample showcases reading and writing operations:
+The following samples demonstrate use of [Authentication](https://cloud.google.com/bigtable/docs/authentication) and [Access Control]((https://cloud.google.com/bigtable/docs/access-control)) for Bigtable. More details on the samples can be found [here](https://googleapis.dev/cpp/google-cloud-bigtable/latest/bigtable-samples-grpc-credentials.html).
```

Comment:
For long links like these consider using [reference style links](https://www.markdownguide.org/basic-syntax/#reference-style-links)